pages/admin/registrant/user/[id].tsx

```diff
@@ -0,0 +1,181 @@
+                        width="full"
+                        display="flex"
+                        marginTop="40px"
+                        paddingRight="20px"
+                        spacing="100px"
+                    >
+                        <VStack
+                            width="400px"
+                            spacing="32px"
+                            marginLeft="40px"
+                            alignSelf="flex-start"
+                        >
+                            <HStack
+                                color={colourTheme.colors.Gray}
+                                width="full"
+                            >
+                                <Icon as={MdPerson} w={8} h={8} />
+                                <VStack
+                                    alignItems="flex-start"
+                                    spacing="-3px"
+                                    w="200px"
+                                >
+                                    <Link>
+                                        {"Participant Information" +
+                                            " (" +
+                                            volunteerName +
+                                            ")"}
+                                    </Link>
+                                </VStack>
+                            </HStack>
+                            <HStack
+                                fontWeight={700}
+                                color={colourTheme.colors.Blue}
+                                width="full"
+                            >
+                                <Icon as={MdDescription} w={8} h={8} />
+                                <Link>Criminal Record Check</Link>
+                            </HStack>
+                        </VStack>
+                        {volunteerData.criminalRecordCheckLink !== null ? (
+                            <FileDownloadCard
+                                filePath={FILE_PATH}
+                                docName={volunteerData.criminalRecordCheckLink}
+                                docApproved={
+                                    volunteerData.criminalCheckApproved
+                                }
+                                participantId={user.id}
+                            />
+                        ) : (
+                            <EmptyState height="200px">
+                                The participant has not uploaded a criminal
+                                record check at this time.
+                            </EmptyState>
+                        )}
+                    </HStack>
+                </VStack>
+            </Wrapper>
+        );
+    }
+    return <Loading />;
+}
+
+/**
+ * getServerSideProps gets the session before this page is rendered
+ */
+export const getServerSideProps: GetServerSideProps = async (context) => {
+    // obtain the next auth session
+    const session = await getSession(context);
+
```

Comment:
kick user if they are not admin

```suggestion
    if (!session) {
        return {
            redirect: {
                destination: "/login",
                permanent: false,
            },
        };
    } else if (!isAdmin(session)) {
        return {
            redirect: {
                destination: "/no-access",
                permanent: false,
            },
        };
    }
```

import isAdmin via `import { isAdmin } from "@utils/session/authorization";`